### ISSUE TITLE
Beep not playing when alerts are triggered

### ISSUE THREAD

**Issue by deathybankai** [ORelio/Minecraft-Console-Client#1036]
Alerts are shown in the console, but no sound is played.
Testing Console.Beep() (Whats used for alerts system) and System.Media.SystemSounds.Beep.Play()
both do not seem to play a sound for me.
Console.WriteLine("\a") seems to work, but I'm not super familiar with C# so I don't know what the most universal option would be or if it's just my system and I need to fix it my side.

Using the latest Dev build. Let me know if you want more info.

**Comment by ORelio** [comment]
Hi,

`System.Media.SystemSounds.Beep.Play()` will likely work on Windows only
`Console.Beep()` *should* auto-detect the correct escape sequence for your terminal
`Console.WriteLine("\a")` I guess it just writes the ASCII Bell character `0x07`

If you could try it on various operating systems and confirm it works better than `Console.Beep()`, please let me know so we change it for `Console.WriteLine("\a")`.

**Comment by deathybankai** [comment]
Found out the issue and sorry it was 100% on my side.

At some point, I turned the system sounds off so all the sounds that play from windows perspective as a system sound such as error alerts, System.Media.SystemSounds.Beep.Play(), and Console.Beep() were muted.
The reason Console.WriteLine("\a") worked was due to the sound registering from the console window rather than directly from the system.

Sadly I'm unable to physically test with MacOS, but I found records of \a working on Linux and MacOS. So it could potentially be used as a more reliable source of the sound if all OS treats it as a sound deriving from the console rather then the system itself.

https://en.wikipedia.org/wiki/Bell_character
https://support.apple.com/lt-lt/guide/terminal/trmladvn/mac
